Objective-C/Objective-C++.sublime-syntax

```diff
@@ -1058,7 +1058,7 @@ contexts:
     - match: '(?={{path_lookahead}}\s*::\s*$)'
       set: [function-definition-params, global-function-identifier]
     - match: '(?=\S)'
-      fail: global-maybe
+      fail: true
```

Comment:
Do you want to `pop`?

Reply:
Yes, otherwise it leads to infinite cycling in case both branches global-maybe fail

Reply:
I will fix to pop in 2-3 hours